week-1/exercises/count.js

```diff
@@ -3,4 +3,10 @@ let count = 0;
 count = count + 1;
 
 // Line 1 is a variable declaration, creating the count variable with an initial value of 0
-// Describe what line 3 is doing, in particular focus on what = is doing
+// Describe what line 3 is doing, in particular focus on what = is doing.
+
+//answer
+//  "=" is an assignment operator in coding for example in line 1 and variable declaration 0 has been assigned to "count" variable.
+//and in line 3 it says add up count with 1 and assign it again to "count" . initial value was 0 , added up with 1 which is 1 and assigned
+// again to ocunt so the new value of "count" is 1 . 
+//count=1 , next itrate would be count=count(==1)+1 => count=2
```

Comment:
Very nice and detailed! Just watch out for your spelling in your technical communication